test/test_helper.exs

```diff
@@ -1 +1,3 @@
 ExUnit.start()
+Aprb.Repo.start_link()
```

Comment:
Because we don't start the app in tests (`--no-start`), we need to do it manually.